contracts/SafePolicyGuard.sol

```diff
@@ -122,7 +140,7 @@ contract SafePolicyGuard is PolicyEngine, ISafeModuleGuard, ISafeTransactionGuar
 
         if (setGuard) removeGuard[safe] = 0;
 
-        return configureOrConfirmPolicy || setGuard || guardRemovalScheduled;
+        return requestOrApplyConfiguration || setGuard || invalidateRootCall;
```

Comment:
Shouldn't the `setGuard` code be removed?

Reply:
Nice catch! Removed 👍🏾